core.coffee

```diff
@@ -34,7 +34,7 @@
 
 	# Utils must support .isString, .isObject, and .isArray
 	# Promise must support Promise.reject, returning a rejected promise
-	return (utils, Promise) ->
+	return (utils, Promise, mimemessage) ->
```

Comment:
Can you add a comment about what is required from `mimemessage` please and add it to the `core.d.ts` types as well please